STUDIO/org.openl.rules.webstudio/src/org/openl/rules/webstudio/web/ErrorsContainer.java

```diff
@@ -0,0 +1,56 @@
+package org.openl.rules.webstudio.web;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public final class ErrorsContainer {
```

Comment:
Javadocs. For what purposes this class was created?

Reply:
Added javadocs